scss/components/_feedback.scss

```diff
@@ -0,0 +1,136 @@
+.feedback {
+  background: $white;
+  padding-top: 125px;
+  padding-bottom: 64px;
+}
+
+.feedback-headline {
+  font-size: 45px;
+  display: flex;
+}
+
+.feedback-underline {
```

Comment:
Consider using nesting here:
```
.feedback-underline {
  padding-bottom: 16px;
  border-bottom: 1px solid gray;
  display: flex;
  align-self: center;
  width: 100px;
  white-space: nowrap;
  margin-left: 137px;

  &-headline {
    margin: -18px 0 0 -3px;
    font-weight: bold;
  }

  &-text {
    margin: 105px 0 0px -101px;
    font-size: 18px;
  }
}
```
Do not use negative margin

Reply:
Fixed